authnid/make_app.py

```diff
@@ -65,3 +78,18 @@ def _make_crl_validator(app):
 
 def _make_token_manager(app):
     app.token_manager = TokenManager(app.config.get("TOKEN_SECRET"))
+
+
+def _apply_request_hooks(app):
+    @app.before_request
+    def before_request():
+        app.db = connect_db(app.db_uri)
```

Comment:
Does this do connection pooling? Or am I prematurely optimizing with my thoughts? My point being... are we spinning up a new DB connection on each request when we don't need to?

Reply:
No connection pooling right now. This was just sort of a rough-out. Right now it is per-request, but we could do it on app spin-up and teardown.